astronomer/providers/sftp/example_dags/example_sftp.py

```diff
@@ -24,14 +227,64 @@
     default_args=default_args,
     tags=["example", "async", "sftp"],
 ) as dag:
+
+    create_ec2_instance = PythonOperator(
+        task_id="create_ec2_instance", python_callable=create_instance_with_security_group
+    )
+
+    # get_instance_details = PythonOperator(
+    #     task_id="get_instance_details", python_callable=get_ec2_instance_details
+    # )
```

Comment:
we can remove commented code if not required

Reply:
Removed it